R/fotmob_helpers.R

```diff
@@ -1,9 +1,3 @@
-
-
-#' @importFrom purrr safely
-#' @importFrom jsonlite fromJSON
-.safely_from_json <- purrr::safely(jsonlite::fromJSON, otherwise = NULL, quiet = TRUE)
```

Comment:
can't do this with `.fromJSON()` (or any function that's defined within the package). this can only be done when calling an external function ("pre-compiled")

Reply:
Yeah good pick up.